WhoIsThatUnitTests/ImageHandlerUnitTests.cs

```diff
@@ -0,0 +1,42 @@
+using System.IO;
+using NUnit.Framework;
+using Shouldly;
+using WhoIsThat.Handlers;
+using Xamarin.Forms;
+
+namespace WhoIsThatUnitTests
+{
+    public class ImageHandlerUnitTests
+    {
+        [Test]
+        public void GetStreamFromUri_ShouldReturn()
```

Comment:
Just noticed, it probably should be like `ShouldReturnCorrectValue` or `ReturnsCorrectValue`?:D